apps/web/pages/[user]/book.tsx

```diff
@@ -110,9 +111,12 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   })[0];
 
   async function getBooking() {
+    console.log(context.query);
     return prisma.booking.findFirst({
       where: {
-        uid: asStringOrThrow(context.query.rescheduleUid),
+        // Find booking for reschedule or taking a seat
+        ...(context.query.rescheduleUid && { uid: asStringOrThrow(context.query.rescheduleUid) }),
+        ...(context.query.bookingId && { id: parseInt(context.query.bookingId as string) }),
```

Comment:
Reuse the `getBooking` function on the booking page. Will either search for a booking with the bookingId for seats or the rescheduled if rescheduling.